content/source/docs/enterprise/saml/configuration.html.md

```diff
@@ -11,19 +11,47 @@ SAML requires the configuration of two parties:
 - The Identity Provider (IdP).
 - The Service Provider (SP), which is also sometimes referred to as Relying Party (RP).
 
-Terraform Enterprise is configured as the Service Provider.
+Private Terraform Enterprise (PTFE) is configured as the Service Provider.
 
-~> For instructions for specific IdPs, see [Identity Provider Configuration](./identity-provider-configuration.html)
+-> **Note:** For instructions for specific IdPs, see [Identity Provider Configuration](./identity-provider-configuration.html).
 
 ## Terraform Enterprise (Service Provider)
 
-Go to `https://<TFE HOSTNAME>/admin/settings/saml` and set the following:
+~> **Important:** Only PTFE users with the site-admin permission can modify SAML settings. For more information about site admins, see [Administering Private Terraform Enterprise][admin].
 
-1. **Single Sign On URL**: specifies the HTTP(S) endpoint on your IdP for single sign-on requests. This value is provided by your IdP configuration.
-2. **Single Log Out URL**:  specifies the HTTP(s) endpoint on your IdP for single logout requests. This value is provided by your IdP configuration. Single Logout is not yet supported.
-3. **Identity Provider Certificate**: Specifies the PEM encoded X.509 Certificate as provided by the IdP configuration.
-4. **Team Attribute Name**: (default: `MemberOf`) Specifies the name of the SAML attribute that determines [team membership](./team-membership.html). The value of this attribute in the SAML assertion must be a string containing a comma-separated list of team names.
-5. **Site Admin Role**: (default: `site-admins`) Specifies the role for site admin access, provided in the list of roles sent in the Team Attribute Name attribute.
+[admin]: ../private/admin/index.html
+
+Go to the SAML section of the site admin pages. You can use the "Site Admin" link in the upper-right user icon menu, or go directly to `https://<TFE HOSTNAME>/app/admin/saml`.
+
+Once there, enter values (or leave the default values) for TFE's SAML settings, then click the "Save SAML Settings" button at the bottom of the page.
```

Comment:
I don't think it makes sense to say leave the default values for a number of the values, so I'd probably just skip that here; it could be noted below, or we could just let there being a default speak for itself.